docs/utils.py

```
@@ -126,3 +139,34 @@ def slugify_header(s: str) -> str:
 	text = re.sub(r'[^\w\- ]', "", s).strip().lower()
 	# Then replace consecutive spaces or dashes with a single dash
 	return re.sub(r'[-\s]+', "-", text)
+
+
+def sanitize_url(url):
```

Ruff (0.8.2)

Missing return type annotation for public function `sanitize_url`

(ANN201)